backend/.env.template

```diff
@@ -1 +1,6 @@
-JWTSECRET=yourjwtsecret
\ No newline at end of file
+JWTSECRET=yourjwtsecret
```

Comment:
⭐ For not committing sensitive secrets (e.g. API keys) unencrypted to the repository.